index.html

```diff
@@ -874,6 +697,250 @@
+          class="mjx-mi"><span class="mjx-char MJXc-TeX-math-I" style=
+          "padding-top: 0.446em; padding-bottom: 0.298em;">b</span></span><span id="MJXc-Node-127"
+          class="mjx-mi"><span class="mjx-char MJXc-TeX-math-I" style=
+          "padding-top: 0.446em; padding-bottom: 0.298em;">i</span></span><span id="MJXc-Node-128"
+          class="mjx-mi"><span class="mjx-char MJXc-TeX-math-I" style=
+          "padding-top: 0.372em; padding-bottom: 0.298em;">t</span></span></span></span></span><span id="MJXc-Node-129"
+          class="mjx-mo" style=
+          "padding-left: 0.333em; padding-right: 0.333em;"><span class=
+          "mjx-char MJXc-TeX-main-R" style=
+          "padding-top: 0.077em; padding-bottom: 0.298em;">=</span></span><span id="MJXc-Node-130"
+          class="mjx-mfrac"><span class="mjx-box MJXc-stacked" style=
+          "width: 2.265em; padding: 0px 0.12em;"><span class="mjx-numerator"
+          style="width: 2.265em; top: -1.447em;"><span id="MJXc-Node-131"
+          class="mjx-msub"><span class="mjx-base" style=
+          "margin-right: -0.003em;"><span id="MJXc-Node-132" class=
+          "mjx-mi"><span class="mjx-char MJXc-TeX-math-I" style=
+          "padding-top: 0.446em; padding-bottom: 0.298em; padding-right: 0.003em;">d</span></span></span><span class="mjx-sub"
+          style=
+          "font-size: 70.7%; vertical-align: -0.219em; padding-right: 0.071em;"><span id="MJXc-Node-133"
+          class="mjx-mrow" style=""><span id="MJXc-Node-134" class=
+          "mjx-mn"><span class="mjx-char MJXc-TeX-main-R" style=
+          "padding-top: 0.372em; padding-bottom: 0.372em;">16</span></span><span id="MJXc-Node-135"
+          class="mjx-mi"><span class="mjx-char MJXc-TeX-math-I" style=
+          "padding-top: 0.446em; padding-bottom: 0.298em;">b</span></span><span id="MJXc-Node-136"
+          class="mjx-mi"><span class="mjx-char MJXc-TeX-math-I" style=
+          "padding-top: 0.446em; padding-bottom: 0.298em;">i</span></span><span id="MJXc-Node-137"
+          class="mjx-mi"><span class="mjx-char MJXc-TeX-math-I" style=
+          "padding-top: 0.372em; padding-bottom: 0.298em;">t</span></span></span></span></span></span><span class="mjx-denominator"
+          style="width: 2.265em; bottom: -0.778em;"><span id="MJXc-Node-138"
+          class="mjx-mn"><span class="mjx-char MJXc-TeX-main-R" style=
+          "padding-top: 0.372em; padding-bottom: 0.372em;">255</span></span></span><span style="border-bottom: 1.3px solid; top: -0.296em; width: 2.265em;"
+          class="mjx-line"></span></span><span style=
+          "height: 2.225em; vertical-align: -0.778em;" class=
+          "mjx-vsize"></span></span></span></span></span></span></span>
+        </p>
+        <div class="note">
+          It is an implementation detail how the frames are rendered to the
+          screen. For example, the implementation may use a grayscale or red
+          channel representation.
```

Comment:
What would be the benefit of introducing a separate video rendering representation that uses a single-channel format (i.e. red)? Is this only for the display rendering or does it include a separate packing format?